tests/phpunit/tests/block-supports/typography.php

```diff
@@ -442,8 +446,7 @@ public function data_generate_font_size_preset_fixtures() {
 
 			'returns value with unsupported unit'        => array(
 				'font_size_preset' => array(
-					'size'  => '1000%',
-					'fluid' => false,
+					'size' => '1000%',
```

Comment:
As above, addressing an omission - here the tests is checking whether the unsupported unit is returned when fluid typography is enabled.